tests/integration/backend_test.py

```diff
@@ -364,7 +365,7 @@ def test_classical(
 
     c.add_classicalexpbox_register(a + b, d)
     c.add_classicalexpbox_register(a - b, d)
-    c.add_classicalexpbox_register(a * b // d, d)
+    # c.add_classicalexpbox_register(a * b // d, d) # division unsupported in QIR
```

Comment:
Can we separate this into another test case, so that we still test it for QASM, but with the QIR part commented out with a FIXME?

Reply:
Will do that

Reply:
(Or just make this line conditional on `language`.)